src/Microsoft.Diagnostics.EventFlow.Inputs.Trace/TraceInput.cs

```diff
@@ -81,6 +81,36 @@ public override void Fail(string message, string detailMessage)
             SubmitEventData(message, TraceEventType.Error);
         }
 
+        public override void Write(object o)
+        {
+            WriteLine(o.ToString());
+        }
+
+        public override void Write(object o, string category)
+        {
+            Write(o);
+        }
+
+        public override void Write(string message, string category)
+        {
+            WriteLine(message);
+        }
+
+        public override void WriteLine(object o)
+        {
+            WriteLine(o.ToString());
+        }
+
+        public override void WriteLine(object o, string category)
+        {
+            WriteLine(o);
+        }
+
+        public override void WriteLine(string message, string category)
+        {
+            WriteLine(message);
+        }
+
         public override void TraceData(TraceEventCache eventCache, string source, TraceEventType eventType, int id, object data)
```

Comment:
For these new overloads we should also add unit tests in https://github.com/Azure/diagnostics-eventflow/blob/master/test/Microsoft.Diagnostics.EventFlow.Inputs.Tests/TraceInputTests.cs. They can be very simple (similar to `TraceInputShouldOverrideWriteLine()`, just to verify that the override works and that the resulting `EventData` object passed down the pipeline is carrying the expected data.